### ISSUE TITLE
mux-r2.8.0 crash with Gradle build tool 3.5.0

### ISSUE THREAD

**Issue by huyto-hitrec** [muxinc/mux-stats-sdk-exoplayer#18]
### Description
When the app tries to instantiate a MuxStatsExoPlayer, it crashes with an AbstractMethodError. This only happens when building with Gradle buildTypes field "debuggable false". This is did not happen in ExoPlayer v2.9.6.

### Code
```
private lateinit var muxStats: MuxStatsExoPlayer

private fun configurePlayer(context: Context) {
        val userAgent = Util.getUserAgent(context, "appName")
        val dataSourceFactory = DefaultDataSourceFactory(context, userAgent)
        val uri = Uri.parse("videoUrl")
        val videoSource = HlsMediaSource.Factory(dataSourceFactory).createMediaSource(uri)
        player.prepare(videoSource)

        val customerPlayerData = CustomerPlayerData()
        customerPlayerData.environmentKey = muxKey
        customerPlayerData.viewerUserId = userId

        val customerVideoData = CustomerVideoData()
        customerVideoData.videoId = videoId
        customerVideoData.videoTitle = title
        customerVideoData.videoSeries = catTitle
        customerVideoData.videoDuration = duration

        // Crashes here
        muxStats = MuxStatsExoPlayer(
            context,
            player,
            PLAYER_NAME,
            customerPlayerData,
            customerVideoData
        )

        val size = Point()
        activity?.windowManager?.defaultDisplay?.getSize(size)
        muxStats.setScreenSize(size.x, size.y)
        muxStats.setPlayerView(playerView.videoSurfaceView)

        player.playWhenReady = true
    }
```

**Comment by huyto-hitrec** [comment]
After more investigation, the cause doesn't seem to be the ExoPlayer version change but rather a change from Gradle build tool (com.android.tools.build:gradle) from version 3.4.2 to 3.5.0. This would make sense why the debuggable flag is causing this crash as the build tool probably changed the way they handle debuggable vs non-debuggable.

**Comment by jsanford8** [comment]
@huyto-hitrec Thanks a lot for the investigation! We'll take a look to see why that would be causing the SDK to crash

**Comment by jsanford8** [comment]
fixed by #20
